README.md

```diff
@@ -29,6 +29,8 @@ Simply check out this repo, set your XCUITest test bundles and run the appropria
 
 :bulb: We also provide [DemoApp](DemoApp/) to demonstrate the test workflow. Click [here](.github/workflows/test.yml) for more details on how to build your XCUITest test pipeline.
 
+To download prebuilt demo apps, run "sh ./scripts/download-test-apps.sh" and the apps will download to a directory called "demo-apps/"
```

Comment:
According to the comment that's in the script, these prebuilts are for simulators.  Just good to have that info here:
```suggestion
To download prebuilt demo apps that target simulators, run "sh ./scripts/download-test-apps.sh" and the apps will download to a directory called "demo-apps/".
```

Reply:
It's actually the bash script that's wrong. I'll update that.

Reply:
Ah :D Well in that case we should add here that the prebuilts are for real devices.